ArmaReforgerServerTool/Forms/Main.cs

```diff
@@ -75,6 +79,20 @@ public Main()
             Mod.GetScenariosForMod("591AF5BDA9F7CE8B");
         }
 
+        /// <summary>
+        /// Retrieves the application's version number.
+        /// </summary>
+        /// <returns>Formatted version string.</returns>
+        private string GetAppVersion()
+        {
+            // Retrieve the version from the assembly metadata using pattern matching
+            return Assembly.GetExecutingAssembly().GetName().Version switch
+            {
+                { } version => $"v{version.Major}.{version.Minor}.{version.Build}.{version.Revision}",
+                _ => "v0.0.0.0" // Fallback version if Version is null
+            };
```

Comment:
I'd prefer throwing on null version. It's the kind of thing that is either set or not, and it's vital for it to be set when released.